### ISSUE TITLE
Toggle for BVH scale

### ISSUE THREAD

**Issue by Erimelowo** [SlimeVR/SlimeVR-Server#1021]
Blender uses a scale 100 times smaller than most other programs. We already have the setting in BVHSettings.java. Need to make it available to change in vrconfig and in the GUI so that it can be more convenient for programs that aren't Blender and don't support scaling like BVHacker does.

**Comment by Makkarob** [comment]
Yes, great idea - I spent several days trying to figure out why in all programs except Blender I just see a bunch of bones at one point, spinning randomly.
